lib/can/include/CANMsg.h

```diff
@@ -8,14 +8,15 @@ class CANMsg : public CANMessage {
  private:
   using CAN_Message::id;
 
+  static mbed_error_status_t defaultCANMsgHandler(CANMsg &msg) {
+    MBED_WARNING(MBED_MAKE_ERROR(MBED_MODULE_APPLICATION, MBED_ERROR_CODE_INVALID_ARGUMENT),
+                 "Invalid key to CANMsgHandlerMap");
+    return MBED_ERROR_CODE_INVALID_ARGUMENT;
+  }
```

Comment:
This function was originally a lambda function, but I'm making it a separate function because of compilation errors with the lambda-function-in-a-header being different across different translation units